extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -459,6 +459,11 @@ export class AmpStoryPage extends AMP.BaseElement {
    *     page.
    */
   setDistance(distance) {
+    // TODO(ccordry) refactor this when pages are managed
+    if (this.isAd()) {
```

Comment:
maybe safer to have it in `getDistance()`?

Reply:
It appears `getDistance` is only called by the media pool. I could maybe put it in `AmpStory.getPageDistanceMapHelper_` but I think we should keep it on the page if possible

Reply:
Yeah, this looks reasonable to me.  Putting it in getDistance may be odd because then the `distance` attribute and the value returned by `getDistance()` would get out of sync.